src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -101,6 +112,16 @@ class PgAutoscaler(MgrModule):
             'name': 'sleep_interval',
             'default': str(60),
         },
+        Option(
+            'autoscale_profile',
+            default='scale-up',
+            type='str',
+            desc='pg_autoscale profiler',
+            long_desc='Determines the behavior of the autoscaler algorithm\
```

Comment:
this multiline string has whitespace too

Reply:
done!